bin/antigen.zsh

```diff
@@ -364,6 +375,8 @@ antigen () {
       branch="${url#*|}"
     fi
     install_or_update=true
+    # Update remote if needed.
+    -antigen-update-remote $clone_dir $url
```

Comment:
I'd put this under a configuration variable. Adding a note on the changelog/release notes about prezto community fork and what means to -update-remote.